src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
```

Comment:
You can nest some styles in this file, I believe that you will find those places.